request-management-api/request_api/models/FOIRawRequests.py

```diff
@@ -245,6 +245,21 @@ def getbasequery(cls, additionalfilter=None, userid=None):
                              literal(None)),
                            ],
                            else_ = FOIRawRequest.requestrawdata['dueDate'].astext).label('duedate')
+        anotherfirstname = case([
+                            (FOIRawRequest.status == 'Unopened' and FOIRawRequest.requestrawdata['anotherInformation']['firstName'] is not None,
```

Comment:
Since Flex team will only see ministry requests, do we need to pull this field from FOIRawRequests. Will this field be needed before that stage?